### ISSUE TITLE
Submit button disabled without submitting the record. 

### ISSUE THREAD

**Issue by saadurrehman-selectquote** [SelectQuoteLabs/SQForm#36]
https://www.loom.com/share/fb06b329882c4cadada5275c5cb36f3f

i have created a state which has individual data , and the reason is that i need to show the changes of zipcode on the onBlur
and i am setting the data into the states(it is the same state which is passed to the SQForm), but at that time it disabled the button which is causing the pain. 

```
 const {dirty} = useSQFormContext();
    const isDisabled = !dirty;
```
i am using SQFormContext to handle this stuff as suggested by Sean. but its not working in this case

**Comment by SeanGroff** [comment]
@saadurrehman-selectquote Thanks for submitting this issue and providing a clear explanation.
Can you send me a link to your feature branch so that I can work with this locally if I cannot recreate this scenario in Storybook?

**Comment by saadurrehman-selectquote** [comment]
sure here is the command and link
git fetch && git checkout store/AZT-1424-fe---crm-address-info-card-util
https://bitbucket.org/SelectQuote/sc2-senior/branch/store/AZT-1424-fe---crm-address-info-card-util?dest=develop#diff

**Comment by SeanGroff** [comment]
@saadurrehman-selectquote I believe this issue is related to issue #41.
I don't believe you are actively working on the feature anymore but if you have some downtime could you update SQForm to version 3.0.2 in your feature branch and see if this resolves the bug reported in this issue?

**Comment by codyarose** [comment]
@saadurrehman-selectquote Can you confirm that this issue is still present? I believe it may have been fixed in #171, which was included in v4.3.6

**Comment by saadurrehman-selectquote** [comment]
@codyarose its fixed, we can close this one
